src/wp-includes/block-template-utils.php

```diff
@@ -432,6 +432,41 @@ function _inject_theme_attribute_in_block_template_content( $template_content )
 	return $template_content;
 }
 
+/**
+ * Parses wp_template content and removes the theme attribute from
+ * each wp_template_part
+ *
+ * @access private
+ * @since 5.9.0
+ *
+ * @param string $template_content serialized wp_template content.
+ *
+ * @return string Updated wp_template content.
+ */
+function _remove_theme_attribute_in_block_template_content( $template_content ) {
```

Comment:
I don't know how we organize things in general in Core but for me, I don't think we should change this file as the added functions are specific to the export behavior and could be moved to the export controller directly (private methods there) or their own file. WDYT?

I see this file as the API to fetch block templates and template parts.

Reply:
We can't test the private method in PHP, but we can make it public.

Also, make sense to have a remove method when we have an inject one.

> I don't know how we organize things in general in Core

I'm also not sure about this. I talked with @noisysocks, and we decided to keep these methods in the utils file for now.

Reply:
Mmm, I don't like that we're kind of changing the role of the "utils" file but I can live with it. We should also do the same in the plugin if we go that road to keep the files similar to easy backports...

Reply:
Done - https://github.com/WordPress/gutenberg/pull/36908

Reply:
Initially, I was planning to have these methods private, but then I got reminded that public API could be helpful for WP CLI and similar tools - https://github.com/WordPress/gutenberg/pull/36559#issuecomment-971789574.

Maybe we can have a separate file for site export methods once API is more mature in the future.